src/2geom/circle.cpp

```diff
@@ -126,6 +126,10 @@ Coord Circle::nearestTime(Point const &p) const {
     return timeAt(p);
 }
 
+Coord Circle::furthestTime(Point const &p) const {
+    return timeAt(p) + rad_from_deg(180);
```

Comment:
This can return anything between 0 and 3*M_PI. It should return something within the range [0, 2*M_PI) instead - the class Angle can be used for this.